model/job-management/src/main/proto/beam_job_api.proto

```diff
@@ -155,6 +159,14 @@ message JobMessage {
   }
 }
 
+message GetJobMetricsRequest {
+  string job_id = 1; // (required)
+}
+
+message GetJobMetricsResponse {
+  org.apache.beam.model.fn_execution.v1.MetricResults metrics = 1;
```

Comment:
If we are adding MetricResults we want it to be uniform across languages. This will make it easier to switch between SDKs and simplify some of code logic in the future.
I don't think that MonitoringInfo will change much in the future, so it should be fine to utilize it here directly.